lib/Backend/amd64/LowererMDArch.cpp

```diff
@@ -523,7 +561,9 @@ LowererMDArch::LowerCallArgs(IR::Instr *callInstr, ushort callFlags, Js::ArgSlot
         callInstr->InsertBefore(argInstr);
         argCount++;
     }
-
+#ifndef _WIN32
+    this->xplatCallArgs.StopRecording();
```

Comment:
if we count them all, we would also count the ones we endup pushing to stack. We are only interested with the ones explicitly located on registers (float and non-float) . The rest, we consider them as non-float.